fl4health/feature_alignment/string_columns_transformer.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from fl4health.feature_alignment.constants import TextFeatureTransformer
+
+
+class StringColumnTransformer(BaseEstimator, TransformerMixin):
```

Comment:
This class doesn't seem to be used anywhere. If you think it's still useful, I'm okay with keeping it, but maybe we name it in a more detailed way. Something like `TextMulticolumnTransformer` maybe?